client/src/pages/EditAccount.js

```diff
@@ -0,0 +1,253 @@
+import React from "react";
+import { useForm } from "react-hook-form";
```

Comment:
I used useForm instead of ant form because I noticed that ant form uses state which results into a lot of re-renders instead of useForm which only uses one.  I can change it to ant form if necessary let me know